File: flutter_app_client/lib/data/item.dart
import 'item_id.dart';

class Item {
  final ItemId id;
  final String title;
  final String subtitle;
  final String description;
  final String imageUrl;
  bool isFavorite;

  Item({
    required this.id,
    required this.title,
    required this.subtitle,
    required this.description,
    required this.imageUrl,
    this.isFavorite = false,
  });
}

File: ApiService/Casino.DAL/Models/Item.cs
﻿namespace Casino.DAL.Models {
    public sealed class Item {
        public int Id { get; set; }
        public string? Title { get; set; }
        public string? Subtitle { get; set; }
        public string? Description { get; set; }
        public string? ImageUrl { get; set; }
    }
}
